### PR TITLE
fix: API merge message is optional

### PR DESCRIPTION
**Pull request by nopcoder**
Fix backward compatibility for merge request

### CODE REVIEW

File: docs/assets/js/swagger.yml [hunk 1]
```diff
@@ -1777,6 +1775,30 @@ paths:
         required: true
         schema:
           type: string
+    get:
```
Comment (arielshaqed) [inline docs/assets/js/swagger.yml:1778]: This seems to be off by one, it incorporates a _previous_ change into this swaggacopy but not _this_ change.  Can you regenerate it before pulling?

**Review by arielshaqed — APPROVED** [review]
Cool, but please keep the js copy of swagger.yml aligned with this PR.

THANKS!

File: docs/assets/js/swagger.yml (hunk 1 above)
Comment (nopcoder) [reply docs/assets/js/swagger.yml:1778]: sure - I was sure the makefile validates they are the same.

Comment (nopcoder) [reply docs/assets/js/swagger.yml:1778]:
```shell
% sha256sum api/swagger.yml docs/assets/js/swagger.yml
9674aad548414c911e7c7d21f22bae22632e9c6e9f7784875549eb74917c34f4  api/swagger.yml
9674aad548414c911e7c7d21f22bae22632e9c6e9f7784875549eb74917c34f4  docs/assets/js/swagger.yml
```

**Pull request merged**